### DISCUSSION TITLE
wifi antenna mod

### DISCUSSION THREAD

**Discussion by mikki8** [roleoroleo/yi-hack-Allwinner-v2#877] [General]
Hi,

i want extend the range of a yi home 1080p.
i can not identify the points to solder it to.
need some help

![grafik](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/76606753/d6db2e73-51f7-49df-bfaa-062ff696cfed)
![grafik](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/76606753/36fdff9d-00e4-4151-b45b-e74809aaf987)

**Comment by roleoroleo** [comment]
I think there are no pads in this pcb.
Take a new picture with a bigger resolution of the blue module.

**Reply by mikki8** [reply]
sure
![grafik](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/76606753/3e41e419-6523-45d3-891a-040ac62f419c)

**Comment by roleoroleo** [comment]
Based on this "datasheet"
https://it.aliexpress.com/item/1005005883802954.html?gatewayAdapt=glo2ita
pin 2 is the antenna.

**Reply by mikki8** [reply]
copy that.
waiting for the adapter.
....
i am not an expert. but shouldn't be there an antenna already ?
or is this the antenna ? 
![grafik](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/76606753/cc9c02fd-ec8d-4c47-9e85-f10aacdd7662)

**Reply by roleoroleo** [reply]
The image is not clear but I think that is the antenna.
The pcb line goes to that block.

**Reply by myhobby2** [reply]
Is it possible that that antenna has been integrated in the USB cable ? I used longer usb cables which resulted in no wifi. Currently the original usb cable is in the camera and then extended.....

**Reply by roleoroleo** [reply]
I have never seen the antenna in the usb cable.
I think is the block in the picture.

**Comment by luckyfreddy** [comment]
Did it work?

**Reply by mikki8** [reply]
hey dude,

i was not sure how to install the new antenna so i created a wifi repeater from an old rasperry pi+wifi stick
